text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/Context.java

```diff
@@ -42,10 +43,11 @@
    * Parses a MiniMessage string using all the settings of this context.
    *
    * @param message the message to parse
+   * @param resolvers additional tag resolvers
    * @return the parsed message
    * @since 4.10.0
    */
-  @NotNull Component parse(final @NotNull String message);
+  @NotNull Component parse(final @NotNull String message, final @NotNull TagResolver@NotNull... resolvers);
```

Comment:
This should be a set of methods that matches the corresponding ones in the main serializer interface.

Reply:
I'm not too familiar with such api designs in java.
Why do you need overloaded methods without varargs here?

```java
ctx.parse("<red>Test");
ctx.parse("<red>bla<place>", Placeholder.parse("place", "bla"));
ctx.parse("<red>bla<array><of><places>", ...);
```
are all covered by the varargs TagResolver method. Or did I miss smth?